tests/delay_handler_test.cc

```diff
@@ -0,0 +1,67 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "delay_handler.h"
+#include <map>
+#include <memory>
+#include <string>
+
+using std::map;
+using std::string;
+using ::testing::HasSubstr;
+
+
+TEST(RequestHandlerTest, createDelayHandler) {
+    std::unique_ptr<RequestHandler> handler(RequestHandler::CreateByName("DelayHandler"));
+    ASSERT_NE(handler, nullptr);
+
+    NginxConfig config;
+    RequestHandler::Status status = handler->Init("/foo", config);
+    EXPECT_EQ(status, RequestHandler::OK);
+}
+
+TEST(DelayHandlerTest, createDelayHandler) {
+    NginxConfig config;
+    std::unique_ptr<RequestHandler> handler(new DelayHandler());
+    ASSERT_NE(handler, nullptr);
+
+    RequestHandler::Status status = handler->Init("/foo", config);
+    EXPECT_EQ(status, RequestHandler::OK);
+}
+
+class DelayHandlerTester : public ::testing::Test {
+    protected:
+        virtual std::unique_ptr<RequestHandler> makeTestDelayHandler(std::string uri_prefix){
+            NginxConfig config;
+            std::shared_ptr<NginxConfigStatement> stmt(new NginxConfigStatement());
+            stmt->tokens_.push_back ("delay");
+            stmt->tokens_.push_back ("1000");
```

Comment:
Maybe we can pass the `"1000"` as a parameter, then we can use that number in the test instead of hardcoding 1000?